### PR TITLE
Initialization buttons are not well aligned - Closes #1553 #1552 #1551 #1550

### LINKED ISSUE #1553: Initialization: Buttons are not well aligned
### Steps to reproduce

1. Login as ready for initialization account `traffic woman skull forest nerve luggage traffic wrestle ensure organ moon century`
1. Go to wallet
1. Click send

### Actual result
Button vertical alignment is a bit different (check  screenshots), it should be uniformal
Text on confirm button is not centrally aligned
'Confirmation in the next step' text is to close to the border
next button is abnormally big :)

image
image

### PR DESCRIPTION
**Pull request by osvaldovega**
### What issue have I solved?

-- #1553 
-- #1552 
-- #1551 
-- #1550

### How have I implemented/fixed it?

Fix css files of the components that have the issue.
Fix the problem of reload the app when user is in request page, there was a problem passing one of the props to the child components.

### How has this been tested?

By interface and by unit test.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

File: src/components/resultBox/resultBox.css [hunk 1]
```diff
@@ -54,7 +48,22 @@
...
-    margin-left: 20px;
+  & footer {
+    padding: 0px;
+    display: flex;
+    justify-content: space-between;
+    position: absolute;
+    bottom: 58px;
+
+    & > div > button {
+      width: 160px !important;
```
Comment (michaeltomasik) [inline src/components/resultBox/resultBox.css]: Please double check if it is possible to avoid `!important`

Comment (osvaldovega) [reply src/components/resultBox/resultBox.css]: totally right, I just fixed

**Comment by osvaldovega** [comment]
after rebase I think this property got erase, it is fixed now

**Comment by Efefefef** [comment]
Thank you Osvaldo

**Pull request merged**